test/spec/upload.js

```diff
@@ -1025,7 +1025,22 @@ describe("tus", function () {
         expect(upload.url).toMatch(/^https:\/\/master\.tus\.io\/files\//);
         console.log("Upload URL:", upload.url); // eslint-disable-line no-console
 
-        validateUploadContent(upload, done);
+        validateUploadContent(upload, function (err) {
+          if (err) {
+            done.fail(err);
+            return;
+          }
+
+          // delete the upload after it was completed
+          upload.abort(true, function (err) {
+            if (err) {
+              done.fail(err);
+              return;
+            }
+
+            validateUploadDeletion(upload, done);
```

Comment:
Can we have a test that verifies that the terminate method sends a DELETE request to the provided upload URL?

Reply:
added